pages/documentation/developer/tutorials/network_modifications_groovy.md

```diff
@@ -0,0 +1,95 @@
+| BranchAdder           | /                | The adder corresponding to the branch to be added. <br/>Should be created beforehand and contain all the required information.                                                                                            |
+| BusOrBusbarSectionId1 | /                | The ID of the busbar (node/breaker network) or of the bus (bus/breaker network) on which the branch should be connected on side 1.                                                                                        |
+| BusOrBusbarSectionId2 | /                | The ID of the busbar (node/breaker network) or of the bus (bus/breaker network) on which the branch should be connected on side 1.                                                                                        |
+| PositionOrder1        | /                | Required in Node/Breaker. The order position of the branch on side 1 to create the corresponding extension. <br/> The branch will be positioned accordingly on the busbar section 1.                                      |
+| PositionOrder2        | /                | Required in Node/Breaker. The order position of the branch on side 2 to create the corresponding extension. <br/> The branch will be positioned accordingly on the busbar section 2.                                      |
+| FeederName1           | Id of the branch | Only in Node/Breaker. An optional name to be put in the ConnectablePosition extension for the side 1 and that will be displayed on the diagrams.                                                                          |
+| FeederName2           | Id of the branch | Only in Node/Breaker. An optional name to be put in the ConnectablePosition extension for the side 1 and that will be displayed on the diagrams.                                                                          |
+| Direction1            | TOP              | Only in Node/Breaker. An optional direction for the side 1 of the branch to be put in the ConnectablePosition extension and that will correspond to the direction of the side 1 of the branch in the Single line diagram. |
+| Direction2            | TOP              | Only in Node/Breaker. An optional direction for the side 1 of the branch to be put in the ConnectablePosition extension and that will correspond to the direction of the side 1 of the branch in the Single line diagram. |
+
+# RemoveLoaBay.groovy
+
+In this script, a load is removed from the initial node/breaker network.
+The load with id LD2 is removed from the voltage level S1VL2.
+
+Before removing the load, the voltage level is:
+
+![](img/network_modifications_groovy/nb_network_s1vl2.png)
+
+After the modification:
+
+![](img/network_modifications_groovy/nb_network_without_ld2.png)
+
+In this method, the only parameter to give is the id of the connectable that we want to remove. It works for injection as well as branches. In this case, the branch and its topology is removed on both sides.
```

Comment:
I would say 
- "for injection**s**"
- "the branch and its topology **are** removed..."